airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/NormalizationActivityImpl.java

```diff
@@ -156,28 +167,49 @@ public NormalizationSummary normalize(final JobRunConfig jobRunConfig,
         () -> context);
   }
 
+  @VisibleForTesting
+  static void activateStrictNormalizationComparisonIfPossible(final IntegrationLauncherConfig destinationLauncherConfig,
+                                                              final FeatureFlags featureFlags) {
+    // Strict comparison was branched from normalization 0.2.25, so we shouldn't apply it if we're
+    // trying to use a newer version of normalization
+    if (featureFlags.nonStrictComparisonNormalizationTag().equals(getNormalizationImageTag(destinationLauncherConfig))) {
```

Comment:
I would remove this check. I think the idea here is to have a feature flag to control which version to use. This checks looks like an extra hurdle that could trip us when setting the override. For example, a normalization bump would disable the feature flag override.